README.md

```diff
@@ -183,113 +190,95 @@ $ ./setup_juno_optee.sh
-	+ fip.bin
-	+ Image
-	+ juno.dtb
+	+ `bl1.bin`
+	+ `fip.bin`
+	+ `Image`
+	+ `juno.dtb`
 
 + Copy OP-TEE binaries on the filesystem(*) located on the external USB key:
 
-	+ user client libraries: libteec.so*
-	+ supplicant: tee-supplicant
-	+ driver modules: optee.ko. optee_armtz.ko
-	+ CA: xtest
-	+ TAs: *.ta
+	+ user client libraries: `libteec.so*`
+	+ supplicant: `tee-supplicant`
+	+ driver modules: `optee.ko optee_armtz.ko`
+	+ CA: `xtest`
+	+ TAs: `*.ta`
 
-+ Connect the USB key (filesystem) on any connector of the rear panel
++ Connect the USB flash drive (containing the filesystem) on any connector of
+  the rear panel
 
-+ Connect a serial terminal (115200, 8, n, 1)
-to the upper 9-pin (UART0) connector.
++ Connect a serial terminal (`115200, 8, n, 1`) to the upper 9-pin (`UART0`)
+  connector.
 
-+ Connect the 12 volt power, then press the red button on the rear panel.
++ Connect the 12V power, then press the red button on the rear panel.
 
 Note:
-The default configuration is to automatically boot a Linux kernel,
-which expects to find a root filesystem on /dev/sda1
-(any one of the rear panel USB ports).
+The default configuration is to automatically boot a Linux kernel, which expects
+to find a root filesystem on `/dev/sda1` (any one of the rear panel USB ports).
 
-(*)Download a minimal filesytem at:
-http://releases.linaro.org/14.02/openembedded/aarch64/
-linaro-image-minimal-genericarmv8-20140223-649.rootfs.tar.gz
+Download a minimal filesytem at &rarr;
+http://releases.linaro.org/14.02/openembedded/aarch64/linaro-image-minimal-genericarmv8-20140223-649.rootfs.tar.gz
 
-UEFI offers a 10 second window to interrupt the boot sequence by pressing
-a key on the serial terminal, after which the kernel is launched.
+UEFI offers a 10 second window to interrupt the boot sequence by pressing a key
+on the serial terminal, after which the kernel is launched.
 
 Once booted you will get the prompt:
 ```
 root@genericarmv8:~#
 ```
 
-#### 4.3.4 Run OP-TEE on ARM Juno
+#### 4.3.5 Run OP-TEE on the Juno board
 Write in the console:
 ```
-root@genericarmv8:~# modprobe optee
+root@genericarmv8:~# modprobe optee_armtz
 root@genericarmv8:~# tee-supplicant &
 ```
 Now everything has been set up and OP-TEE is ready to be used.
 
-#### 4.3.5 Known problems and limitations
-ARM Juno could be sensitive on the USB memory type (filesystem)
-Recommendation: Use USB memory 3.0 (ext3/ext4 filesystem)
+#### 4.3.6 Known issues and limitations
+* `bl30.bin` (SCP) and `bl33.bin` (UEFI) are not available on previous
+  download location. This forces the end user to get this by himself (download,
+  compile etc). We are working with sorting out this issue and once done, we
+  will start using repo manifests for Juno also.
+* Not all USB flash drives seems to work, so we recommend using USB memory 3.0
+  formatted with an ext3/ext4 filesystem
 
 ---
-### 4.4 QEMU
-
-Please refer to section [6. repo manifests](#6-repo-manifests).
+### 4.4 STMicroelectronics boards
```

Comment:
As STM boards are not "opened" boards, I wonder whether we should remove this section or not. Or we could just leave what's below and how to compile.